### PR TITLE
Bug 1842216 exclude main_remainder_v4 from telemetry_missing_columns_v3

### PR DESCRIPTION
**Pull request by lelilia**
Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

### CODE REVIEW

**Comment by lelilia** [comment]
the view-validation does not succeed .. any ideas?

**Comment by lelilia** [comment]
after excluding the view the ci is only failing due to adjust which I didn't touch in this PR

File: sql/moz-fx-data-shared-prod/monitoring_derived/telemetry_missing_columns_v3/query.sql [hunk 1]
```diff
@@ -20,7 +20,7 @@ extracted AS (
     DATE(submission_timestamp) = @submission_date
     -- https://cloud.google.com/bigquery/docs/querying-wildcard-tables#filtering_selected_tables_using_table_suffix
     -- exclude pings derived from main schema to save on space, 300GB vs 3TB
-    AND _TABLE_SUFFIX NOT IN ('main_v4', 'saved_session_v4', 'first_shutdown_v4')
+    AND _TABLE_SUFFIX NOT IN ('main_v4', 'saved_session_v4', 'first_shutdown_v4', 'main_remainder_v4')
```
Comment (relud) [inline sql/moz-fx-data-shared-prod/monitoring_derived/telemetry_missing_columns_v3/query.sql:23]: this also needs to include saved_session_remainder_v4 and first_shutdown_remainder_v4

**Comment by lelilia** [comment]
CI validate-views is failing 
```
sql/moz-fx-data-shared-prod/adjust/adjust_kpi_deliverables/view.sql ERROR
 [{'code': 404, 'errors': [{'message': 'Not found: Dataset moz-fx-data-shared-prod:adjust was not found in location US', 'domain': 'global', 'reason': 'notFound'}], 'response': {'headers': {'alt-svc': 'h3=":443"; ma=2592000,h3-29=":443"; ma=2592000', 'cache-control': 'private', 'content-encoding': 'gzip', 'content-type': 'application/json; charset=UTF-8', 'date': 'Tue, 11 Jul 2023 10:45:04 GMT', 'server': 'ESF', 'transfer-encoding': 'chunked', 'vary': 'Origin, X-Origin, Referer', 'x-content-type-options': 'nosniff', 'x-frame-options': 'SAMEORIGIN', 'x-xss-protection': '0'}}, 'message': 'Not found: Dataset moz-fx-data-shared-prod:adjust was not found in location US'}]
```
but that is not related to anything I touched and seems to be an ongoing issue

**Review by relud — APPROVED** [review]
r+wc

**Pull request merged**